.github/workflows/push-to-prod.yml

```diff
@@ -18,15 +18,21 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v2
-      - name: Random sleep (balena API workaround)
```

Comment:
We need to keep these random sleep bits. Without this the balena API was often failing